src/main/scala/tech/cryptonomic/conseil/config/ConseilConfig.scala

```diff
@@ -0,0 +1,52 @@
+package tech.cryptonomic.conseil.config
+
+import com.typesafe.scalalogging.LazyLogging
+import scala.concurrent.duration.FiniteDuration
+
+object ConseilConfig extends LazyLogging {
+
+  final case class ServerConfiguration(hostname: String, port: Int)
+
+  final case class LorreConfiguration(
+    sleepInterval: FiniteDuration,
+    feeUpdateInterval: Int,
+    purgeAccountsInterval: Int,
+    numberOfFeesAveraged: Int
+  )
+
+  final case class BatchFetchConfiguration(
+    accountConcurrencyLevel: Int,
+    blockOperationsConcurrencyLevel: Int
+  )
+
+  /** configurations related to tezos-node network node calls */
+  final case class TezosRequestsConfiguration(
```

Comment:
As we add more configurations to different blockchains, are we just going to add them all to this file? Would it make sense to split up chain specific configs?

Reply:
I think it will be more sensible to split different networks when they add up...
Though I would advice to wait for the appropriate time, when the separation of the pieces will be clearer... at the moment it's not very clear-cut how and which of these configurations might be shared between platforms or not.